docs/standard/frameworks.md

```diff
@@ -5,6 +5,8 @@ ms.date: 12/12/2023
 ms.service: dotnet
 ms.custom: updateeachrelease
 ms.subservice: standard-library
+f1_keywords:
+- http://schemas.microsoft.com/developer/msbuild/2003#TargetFramework
```

Comment:
@ghogen I thought the f1_keywords were used to match what the editor sends to an article. Is this the string that the editor sends?

Reply:
Yes, we demonstrated that by reviewing the keywords that are coming in, but I wanted to make this change to confirm that everything will work as expected. If successful, I think we could implement F1 for a lot of MSBuild properties, item lists, etc..